src/common/parse-util.c

```diff
@@ -0,0 +1,43 @@
+#include "parse-util.h"
+#include <errno.h>
+#include <inttypes.h>
+#include <limits.h>
+#include <stdbool.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
```

Comment:
```suggestion
#include "parse-util.h"

#include <errno.h>
#include <limits.h>
#include <stdlib.h>
#include <string.h>
```
`inttypes.h` and `stdbool.h` is already included in `parse-util.h`. And I don't see any usage of `stdio.h`, but maybe I missed it?

Reply:
Done